pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -370,7 +398,10 @@ def default_compilation_pass(
         # https://cqcl.github.io/pytket-qiskit/api/index.html#default-compilation
         # Edit this docs source file -> pytket-qiskit/docs/intro.txt
         if optimisation_level == 0:
-            if self._standard_gateset:
+            if self._supports_rz:
+                # If the Rz gate is unsupported then the rebase should be skipped
+                # This prevents an error when compiling to the stabilizer backend
```

Comment:
This check is to ensure that the rebase is skipped when compiling to the remote stabilizer simulator. Basically the simulator does not support the Rz gate meaning no TK1 replacement can be found for the gateset.

Skippping the rebase (as was done before this PR as well) prevents a test failure [here](https://github.com/CQCL/pytket-qiskit/blob/develop/tests/backend_test.py#LL1109C1-L1109C1)